src/test/java/movieinformation/integrationtest/MovieInformationRestApiIntegrationTest.java

```diff
@@ -85,4 +91,31 @@ class MovieInformationRestApiIntegrationTest {
                 .andExpect(MockMvcResultMatchers.status().isNotFound())
                 .andReturn().getResponse().getErrorMessage();
     }
+
+    //Create機能のIntegrationTest
+    @Test
+    @DataSet(value ="datasets/movieData.yml")
+    @ExpectedDataSet(value ="datasets/insert_movieData.yml", ignoreCols = "id")
+    @Transactional
+    public void 新規の映画がDBに登録される事とステータスコード201が返ってくる事()throws Exception{
```

Comment:
ちょっとだけ日本語が変になっちゃってますね！
```suggestion
    public void 新規の映画がDBに登録されるとステータスコード201が返ってくる事()throws Exception{
```